trino/mapper.py

```diff
@@ -175,37 +176,38 @@ class ArrayValueMapper(ValueMapper[List[Optional[Any]]]):
     def __init__(self, mapper: ValueMapper[Any]):
         self.mapper = mapper
 
-    def map(self, values: List[Any]) -> Optional[List[Any]]:
-        if values is None:
+    def map(self, value: Optional[List[Any]]) -> Optional[List[Any]]:
+        if value is None:
             return None
-        return [self.mapper.map(value) for value in values]
+        return [self.mapper.map(v) for v in value]
 
 
 class MapValueMapper(ValueMapper[Dict[Any, Optional[Any]]]):
     def __init__(self, key_mapper: ValueMapper[Any], value_mapper: ValueMapper[Any]):
         self.key_mapper = key_mapper
         self.value_mapper = value_mapper
 
-    def map(self, values: Any) -> Optional[Dict[Any, Optional[Any]]]:
-        if values is None:
+    def map(self, value: Any) -> Optional[Dict[Any, Optional[Any]]]:
+        if value is None:
             return None
         return {
-            self.key_mapper.map(key): self.value_mapper.map(value) for key, value in values.items()
+            self.key_mapper.map(k): self.value_mapper.map(v) for k, v in value.items()
         }
 
 
 class RowValueMapper(ValueMapper[Tuple[Optional[Any], ...]]):
-    def __init__(self, mappers: List[ValueMapper[Any]], names: List[str], types: List[str]):
+    def __init__(self, mappers: List[ValueMapper[Any]], names: List[Optional[str]], types: List[str]):
         self.mappers = mappers
         self.names = names
         self.types = types
 
-    def map(self, values: List[Any]) -> Optional[Tuple[Optional[Any], ...]]:
-        if values is None:
+    def map(self, value: Optional[List[Any]]) -> Optional[Tuple[Optional[Any], ...]]:
+        if value is None:
             return None
+        # TODO: Fix typing for self.names
```

Comment:
another TODO

Reply:
oh, this is easy to fix in the commit which adds typing for types.py.

Let me fix and remove TODO instead. I left it as a note for myself but forgot.